package-lock.json

```diff
@@ -2352,7 +2352,7 @@
         },
         "commander": {
             "version": "2.8.1",
-            "resolved": "https://registry.npmjs.org/commander/-/commander-2.8.1.tgz",
```

Comment:
Oof that https to http thing popped back up for ya. You gotta clear your npm cache or whatever. Let me know if you need help finding the fix